src/models.py

```diff
@@ -98,8 +98,12 @@ def __init__(self, model, client):
                 if callable(attr):
                     self.should_index = attr
                 else:
-                    raise AlgoliaIndexError('{} should be a callable.'.format(
-                        self.should_index))
+                    should_index_attr_name = self.should_index
+                    if isinstance(attr, bool):
```

Comment:
at this point we don't know the type of `attr`, since `attr = getattr(model, self.should_index)`, then it could be a Django field or an unbound method (if I'm right). Hence you cannot typecheck, and just need to keep the line below: `self.should_index = lambda instance: getattr(instance, should_index_attr_name) is True`